c/src/ml-api-service-private.h

```diff
@@ -125,6 +150,16 @@ int _ml_service_query_release_internal (ml_service_s *mls);
  */
 const gchar * _ml_service_get_json_string_member (JsonObject *object, const gchar *member_name);
 
+/**
+ * @brief Invoke ml-service event for new data.
```

Comment:
This comment looks ambiguous. Invoke event?

Reply:
This function is responsible for generating an ML service event when new data is received and calling the callback function registered for that event. I will modify it with an appropriate brief.
Thank you. :pray:

Reply:
Updated, Thank you.